db/client.go

```diff
@@ -0,0 +1,45 @@
+package db
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log"
+
+	"github.com/jbarzegar/ron-mod-manager/config"
+	"github.com/jbarzegar/ron-mod-manager/ent"
+	"github.com/jbarzegar/ron-mod-manager/ent/archive"
+)
+
+var _Client *ent.Client = nil
```

Comment:
Go has an idiom around omitting values when using the default value, so you could safely leave this as :

```go
var _Client *ent.Client
```

Reply:
I'd also say, if you don't need to access `_Client` outside of this package, just set it to `client` so that it's unexported.

Also, in general, Go doesn't like `_` prefixes.